lib/gemstash/logging.rb

```diff
@@ -1,7 +1,12 @@
 # frozen_string_literal: true
 
 require "logger"
-require "puma/events"
+
+begin
+  require "puma/log_writer" # Puma 6
+rescue LoadError
+  require "puma/events"
```

Comment:
I wonder, when you put in ~> 6.1 in gemspec, will anyone ever run an older Puma with this code?